src/content/concepts/configuration.md

```diff
@@ -4,25 +4,26 @@ sort: 6
 contributors:
 - TheLarkInn
 - simon04
+- byzyk
 ---
 
 You may have noticed that few webpack configurations look exactly alike. This is because **webpack's configuration file is a JavaScript file that exports an object.** This object is then processed by webpack based upon its defined properties.
 
 Because it's a standard Node.js CommonJS module, you **can do the following**:
 
-* import other files via `require(...)`
-* use utilities on npm via `require(...)`
-* use JavaScript control flow expressions i. e. the `?:` operator
-* use constants or variables for often used values
-* write and execute functions to generate a part of the configuration
+- import other files via `require(...)`
```

Comment:
Please use title case in lists, same as #L24

Reply:
oops, seems like prettier formatted it automatically. will update tom